README.md

```diff
@@ -1,16 +1,42 @@
+COSI repos and images:
```

Comment:
Start with a headline, which is the project name

Reply:
These links are supposed to emulate a nav bar like feature to provide easier navigation between repos. That would naturally go above the headline. If the navigation bar like feature went below the headline then it might imply that the items in the navigation bar like feature were a part of the item referenced in the headline. If you really want the headline above the navigation bar, that's fine with me. I will update it in next PR mentioned below.

Reply:
I can update in this pr actually and make new pr's for the other 4 README.md.

Reply:
or I can remove these navigation links altogether.